calour/heatmap/plotgui_qt5.py

```diff
@@ -81,10 +81,10 @@ def _update_info_labels(self, sid, fid, abd):
         self.app_window.w_sid.setText(str(sid))
         sample_field = str(self.app_window.w_sfield.currentText())
         self.app_window.w_sfield_val.setText(
-            str(self.exp.sample_metadata[sample_field][self.current_select[0]]))
+            str(self.exp.sample_metadata[sample_field].iloc[self.current_select[0]]))
```

Comment:
why is this change?

Reply:
otherwise, if the index is an int (such as happens in ms gnps table), it returns an error (looks for the number in the index instead of line number....)